### ISSUE TITLE
Copying an Experience Page with incompatible content type

### ISSUE THREAD

**Issue by NPavao** [adampengh/bloomreach-content-saas-ui-extensions#12]
When copying an Experience Page from one channel to another, if the content type does not match, the API responds with 400 and refuses the PUT. 

Improvement: Capture this 400 refusal and give the user an option to delete the destination Experience Page before trying to PUT again. This should allow the Experience Page copy.

**Comment by adampengh** [comment]
@NPavao I don't think this is possible as you cannot delete a Page. You can only dissociate an experience page from a project, but you cannot delete an existing page in Core.
https://documentation.bloomreach.com/content/reference/deletepage_1
